src/dataflow/concatenate_h5mu/config.vsh.yaml

```diff
@@ -1,7 +1,7 @@
 name: concatenate_h5mu
 namespace: "dataflow"
 description: |
-  Concatenates several uni-modal samples in .h5mu files into a single file.
+  Concatenates several uni- and multi-modal samples in .h5mu files into a single file.
```

Comment:
I'm nitpicking at this point, but I think the following minor adjustment should also make it more clear that you can even mix multimodal with unimodal samples? And that the concatenation is done observation-wise, not feature-wise. 
```suggestion
  Concatenate observations from samples in several (uni- and/or multi-modal) MuData files into a single file.
```